integrations/docker/images/chip-build-k32w/Dockerfile

```diff
@@ -5,6 +5,7 @@ FROM connectedhomeip/chip-build:${VERSION}
 RUN set -x \
     && apt-get update \
     && DEBIAN_FRONTEND=noninteractive apt-get install -y wget unzip \
+    && pip3 install pycrypto && pip3 install pycryptodome \
```

Comment:
Should I create another PR only with the above change (pip3 install pycrypto && pip3 install pycryptodome)? Then I believe I need to talk to Andrei Litvin to update the Docker image on the repo because I don't have push access. Is this the correct approach?

Reply:
I mean, we can't add workflows which break CI, so if the workflow requires Docker changes it should be added after the Docker image has been updated. If you can skip signing step for now so that the workflow completes without an error it's probably the fastest way to have your platform verified in CI.